src/lib/mina/fetch.ts

```diff
@@ -769,7 +805,6 @@ async function fetchActions(
       );
     }
   });
-  addCachedActions({ publicKey, tokenId }, actionsList, graphqlEndpoint);
```

Comment:
Is it required to remove addCachedActions? If yes, could you please explain the reason

Reply:
Yes good callout.  I forgot to replace this after moving some code around.

Reply:
Oh, I did not forget.  `addCachedActions` is still present in this commit, you can see it on line 738.